src/main/resources/org/sagebionetworks/web/client/widget/entity/tabs/ChallengeTabViewImpl.ui.xml

```diff
@@ -3,16 +3,14 @@
 	xmlns:g="urn:import:com.google.gwt.user.client.ui"
 	xmlns:b="urn:import:org.gwtbootstrap3.client.ui"
 	xmlns:bh="urn:import:org.gwtbootstrap3.client.ui.html">
-	<bh:Div>
-		<b:Row ui:field="adminTabContainer" addStyleNames="challengeTabContent" >
```

Comment:
In general swap Bootstrap3 rows/columns for FlowPanel because we aren't using the grid/responsive features. 

I did this primarily to simplify dealing with side margins